x-pack/plugins/saved_objects_tagging/public/management/components/table.tsx

```diff
@@ -187,6 +191,10 @@ export const TagTable: FC<TagTableProps> = ({
       items={tags}
       pagination={tablePagination}
       sorting={sorting}
+      tableCaption={i18n.translate('xpack.savedObjectsTagging.management.table.columns.caption', {
+        defaultMessage: 'Tags',
+      })}
+      rowHeader="name"
```

Comment:
By default, EUI forces column headers for tables but very often there's also either an explicit or implicit row header for each row. If there's anything along the lines of a `name` column, that'll usually be the row header as well.